packages/teams-js/src/public/clipboard.ts

```diff
@@ -82,7 +82,11 @@ export namespace clipboard {
     if (!isSupported()) {
       throw errorNotSupportedOnPlatform;
     }
-    if (isHostClientMobile() || GlobalVars.hostClientType === HostClientType.macos) {
+    if (
+      isHostClientMobile() ||
+      GlobalVars.hostClientType === HostClientType.macos ||
```

Comment:
Can you remind me why we need to do this for desktop and macos? In general, it's good to avoid adding platform specific checks like this if we can because it can be hard to maintain or change in the future once it is checked into TeamsJS and removed from `@beta`. Is there some way we could call `sendAndHandleSdkError` and then look at the contents of what is returned and decide whether or not to call `base64ToBlob` based on just that without looking at `hostClientType`?

Reply:
I have updated as you mentioned. Now depending on the response received we decide to call `base64ToBlob` without relying on `hostClientType`